lib/lolapi.js

```diff
@@ -148,7 +148,14 @@
             if (options.endIndex) {
                 historyOptions += '&endIndex=' + options.endIndex;
             }
+            if (options.beginTime) {
```

Comment:
since beginTime 0 would result in the same response as if the option is not set i don't see the point of the suggested check